### ISSUE TITLE
Wrong responsive flex value applied.

### ISSUE THREAD

**Issue by Janjuks** [angular/flex-layout#516]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Screen size is determined correctly on page load

#### What is the current behavior?
On page load, screen size is determined incorrectly. After resizing window back and forth, it is determined correctly.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-flex-layout-seed-5keqb5
Open preview window in new tab with 1920px width. Initially screen size determined is wrong.
https://prnt.sc/hfyob1
After making window smaller and back to original size, it is then correct.
https://prnt.sc/hfyon4

TL;DR - `fxflex.gt-lg` doesn't get applied on maximised window on 1920x1080 resolution

The issue appears when `fxflex.gt-lg` is used together with `fxFlex.lt-xl`

**Comment by ThomasBurleson** [comment]
You are using conflicting responsive APIs

Instead of 

```html
<div fxFlex.gt-lg="50" fxFlex.lt-xl="100" class="sec1">
```

try
```html
<div fxFlex.gt-lg="50" fxFlex="100" class="sec1">
```

> Will fix.
